pom.xml

```diff
@@ -93,7 +93,7 @@
       <dependency>
         <groupId>com.google.cloud</groupId>
         <artifactId>google-cloud-conformance-tests</artifactId>
-        <version>0.0.11</version>
+        <version>0.1.1</version>
```

Comment:
Does this verify the change?

Reply:
Yeah, in fact the failures caused by it is what brought the bug to my attention (https://github.com/googleapis/java-storage/pull/510)